gcp-cloud-run-fizzbuzz-example/massdriver.yaml

```diff
@@ -39,6 +42,19 @@ params:
           title: Max Instances
           description: The maximum number of instances to run at once.
           default: 5
+    api:
+      type: object
+      title: API Configuration
+      required:
+        - path
+      properties:
+        path:
+          type: string
+          title: Path
+          description: Last path segment of this API resource. Entering {proxy+} will respond to all paths.
+          pattern: "((^[a-zA-Z0-9-_]+$)|(^\\{[a-zA-Z0-9-_]+\\}$)|(^\\{proxy\\+\\}$))"
```

Comment:
Is this copy/pasta from lambda? {proxy+} and {customerId} are API gateway specific things. Is it supported in GCP?

Reply:
Yes, wasn't aware it was lambda specific. I'll double check gcp docs and edit